### PR TITLE
Postgres with Flyway initial configuration

### LINKED ISSUE #7: Write the queries to create the tables related to objects we need to persist
Related to user story #6

### PR DESCRIPTION
**Pull request by augustocmleal**
## What this PR changes/adds

_Configure the project to create all the required tables on the PostgreSQL database._

## Linked Issue(s)

Closes #7 - #8 - #9

### CODE REVIEW

File: extensions/postgres/src/main/java/org/eclipse/edc/catalog/Main.java
Comment (schoenenberg) [inline extensions/postgres/src/main/java/org/eclipse/edc/catalog/Main.java:1]: I think we can remove the `Main.java`.

File: extensions/postgres/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    `java-library`
+    id("org.flywaydb.flyway") version "9.16.3"
+}
+
+flyway {
+    url = "jdbc:postgresql://localhost:5432/federated_catalog"
+    driver= "org.postgresql.Driver"
+    user = "yourLocalBdUser"
+    password = "yourLocalBdPassword"
+}
```
Comment (schoenenberg) [inline extensions/postgres/build.gradle.kts:11]: We should make this configurable through the EDC configuration mechanisms.

**Review by schoenenberg — COMMENTED** [review]
I think we should aim for making this an EDC Extension, like [postgres-flyway](https://github.com/sovity/edc-extensions/tree/main/extensions/postgres-flyway).

I am not sure if we should merge it like it is right now and do the EDC Extension in a separate PR. Or do all of it in this PR. @dhommen opinions?

**Comment by augustocmleal** [comment]
Declined Pull Resquer because we will do the Connector configuration First.

**Pull request closed without merging**